docs/src/inputs/dropdownDeprecated/dropdownDeprecated.jsx

```diff
@@ -1285,94 +1285,90 @@ class ModulesDropdown extends React.Component {
                         </Header>
 
                         <Grid>
-                            <Grid.Row stackable>
-                                <Grid.Column width={6} tablet={4} laptop={2}>
-                                    <Dropdown button buttonColor="alert" collapseMenuOnChange text="Alert" style={{ maxWidth: '200px', width: '100%' }}>
+                            <Grid.Column width={6} tablet={4} laptop={2}>
```

Comment:
We're missing the spacing between the Dropdown examples, is that expected?

![image](https://user-images.githubusercontent.com/8116485/125362967-6314a000-e335-11eb-90ac-e5e2f3a2f183.png)

Reply:
Yeah, thanks for checking. Yeah, I think i am okay with that out of time and because it's deprecated.

Reply:
No worries then, I agree with you, thanks.

Reply:
Is it really hard to give them a little bit of margin?